lib/plugins/aws/lib/naming.js

```diff
@@ -406,6 +406,15 @@ module.exports = {
     )}EventSourceMappingSQS${this.normalizeNameToAlphaNumericOnly(queueName)}`;
   },
 
+  // MSK
+  getMSKEventLogicalId(functionName, clusterName, topicName) {
+    return `${this.getNormalizedFunctionName(
+      functionName
+    )}EventSourceMappingMSK${this.normalizeNameToAlphaNumericOnly(
+      clusterName
+    )}${this.normalizeNameToAlphaNumericOnly(topicName)}`;
```

Comment:
We have `255` max chars length. It'll be nice to probably secure we do not overflow that.

I think we can assume that normalizedFunction name, won't exceed 96 chars (at least such constraints will be guaranteed after we implement https://github.com/serverless/serverless/issues/7056).

Them maybe we should set max of `79` chars for both normalized `clusterName` and `topicName` (I would probably just trim it, if it exceeds, with appropriate comment)

Reply:
Good catch - I followed your suggestion here